runners/flink/runner/src/test/java/org/apache/beam/runners/flink/streaming/UnboundedSourceITCase.java

```diff
@@ -1,208 +0,0 @@
-/*
```

Comment:
Would it make sense to keep an ITCase, e.g. to test parallel execution of the sources?